eclair-core/src/main/resources/reference.conf

```diff
@@ -60,7 +60,7 @@ eclair {
     trampoline_payment = disabled
     keysend = disabled
   }
-  override-features = [ // optional per-node features
+  override-init-features = [ // optional per-node features
```

Comment:
Can you also update `Configure.md` with that new parameter name?

Reply:
Done